Source/Core/EllipseGeometry.js

```diff
@@ -175,15 +175,15 @@ define([
                         }
                     }
 
-                    if (vertexFormat.binormal) {
-                        binormal = Cartesian3.normalize(Cartesian3.cross(normal, tangent, binormal), binormal);
-                        binormals[i] = binormal.x;
-                        binormals[i1] = binormal.y;
-                        binormals[i2] = binormal.z;
+                    if (vertexFormat.bitangent) {
+                        bitangent = Cartesian3.normalize(Cartesian3.cross(normal, tangent, bitangent), bitangent);
+                        bitangents[i ] = bitangent.x;
+                        bitangents[i1] = bitangent.y;
+                        bitangents[i2] = bitangent.z;
                         if (extrude) {
-                            binormals[i + bottomOffset] = binormal.x;
-                            binormals[i1 + bottomOffset] = binormal.y;
-                            binormals[i2 + bottomOffset] = binormal.z;
+                            bitangents[i  + bottomOffset] = bitangent.x;
```

Comment:
I appreciate the eye for code formatting and cleanliness, but as a general rule we try to rely on auto-formatting (Webstorm CTRL-SHIFt-L) for blocks of code because it makes it more consistent across the code base as a whole. Unless we starting formatting array lookups like this everywhere, it's not ideal to only do it in a few places (or only have certain devs do it). A good mantra we try to follow is that the code should look like it was all written by one person, even though over 100 people have contributed so far.

That being said, in the future I'm hoping we can move to eslint and you that to enforce common style throughout the codebase.  These types of rules would then become trivial to add or enforce.